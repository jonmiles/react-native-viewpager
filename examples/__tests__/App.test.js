import React from "react";
import renderer from "react-test-renderer";

jest.mock("Dimensions");
import App from "../App";

describe("examples/App", () => {
  it("renders", () => {
    const rendered = renderer.create(<App />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
