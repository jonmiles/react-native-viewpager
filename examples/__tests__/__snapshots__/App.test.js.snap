// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples/App renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "height": 100,
      "justifyContent": "center",
      "marginTop": 20,
    }
  }
>
  <View
    style={
      Object {
        "height": 100,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={undefined}
        onMoveShouldSetResponder={[Function]}
        onMoveShouldSetResponderCapture={[Function]}
        onResponderEnd={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderReject={[Function]}
        onResponderRelease={[Function]}
        onResponderStart={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        onStartShouldSetResponderCapture={[Function]}
        style={
          Object {
            "flex": 1,
            "left": 0,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "bottom": 0,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": 100,
              },
              Object {
                "left": 0,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "justifyContent": "space-around",
                  "width": 100,
                },
                Object {
                  "backgroundColor": "red",
                },
              ]
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "fontSize": 36,
                    "padding": 10,
                    "textAlign": "center",
                  },
                  Object {
                    "backgroundColor": "red",
                    "color": "white",
                  },
                ]
              }
            >
              RED
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#AAA",
                    "fontSize": 70,
                    "fontWeight": "900",
                    "paddingRight": 0,
                    "textAlign": "center",
                  },
                  Object {
                    "backgroundColor": "red",
                    "color": "white",
                  },
                ]
              }
            >
              Richard
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "fontSize": 18,
                    "padding": 10,
                    "textAlign": "center",
                  },
                  Object {
                    "backgroundColor": "red",
                    "color": "white",
                  },
                ]
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontWeight": "900",
                  }
                }
              >
                Richard 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Of 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                York 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Gave 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Battle 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                In 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Vain 
              </Text>
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "bottom": 0,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "top": 0,
                "width": 100,
              },
              Object {
                "left": 100,
              },
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "justifyContent": "space-around",
                  "width": 100,
                },
                Object {
                  "backgroundColor": "orange",
                },
              ]
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "fontSize": 36,
                    "padding": 10,
                    "textAlign": "center",
                  },
                  Object {
                    "backgroundColor": "orange",
                    "color": "white",
                  },
                ]
              }
            >
              ORANGE
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "color": "#AAA",
                    "fontSize": 70,
                    "fontWeight": "900",
                    "paddingRight": 0,
                    "textAlign": "center",
                  },
                  Object {
                    "backgroundColor": "orange",
                    "color": "white",
                  },
                ]
              }
            >
              Of
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "fontSize": 18,
                    "padding": 10,
                    "textAlign": "center",
                  },
                  Object {
                    "backgroundColor": "orange",
                    "color": "white",
                  },
                ]
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Richard 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontWeight": "900",
                  }
                }
              >
                Of 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                York 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Gave 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Battle 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                In 
              </Text>
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
              >
                Vain 
              </Text>
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
